java-tools/OsmAndServer/src/main/java/net/osmand/server/api/searchtest/dto/DatasetRecord.java

```diff
@@ -0,0 +1,25 @@
+package net.osmand.server.api.searchtest.dto;
```

Comment:
Files are too small all package could fit several files, as we store in the project. Move it to repository RunStatus -> RunRepository, TestCase,  Function -> TestCaseRepository...